base/pkg/pkg.jl

```diff
@@ -203,8 +203,12 @@ pin(pkg::AbstractString, ver::VersionNumber) = cd(Entry.pin,pkg,ver)
 Update the metadata repo – kept in `Pkg.dir("METADATA")` – then update any fixed packages
 that can safely be pulled from their origin; then call `Pkg.resolve()` to determine a new
 optimal set of packages versions.
+
+    update(pkgs...)
```

Comment:
I think these should be separate docstrings for the docsystem to handle them properly - the new signature should be added to the rst version of the docs (kinda messy at the moment) so `make docs` populates the rest of it